block/retriever.go

```diff
@@ -90,6 +90,9 @@ func (m *Manager) processNextDABatch(ctx context.Context, daMetaData *da.DASubmi
 			if block.Header.Height != m.store.NextHeight() {
 				continue
 			}
+			if err := m.validateBlock(block, batch.Commits[i]); err != nil {
+				return fmt.Errorf("validate block: height: %d: %w", block.Header.Height, err)
```

Comment:
i think its better to continue here

Reply:
should you really get invalid blocks from the DA though?
Keeping it returning an error is essentially what we already do, making it continue is a change in functionality